blueprints/component/index.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-env node */
+// Note - this is the exact same as ember's normal component index.js
+
+var stringUtil         = require('ember-cli-string-utils');
```

Comment:
Lines 4-8 don't follow this own style guide ;-) https://github.com/softlayer/ember-style-guide/blob/master/javascript.md#whitespace